src/Scripting/CSharpTest/CommandLineRunnerTests.cs

```diff
@@ -947,5 +947,48 @@ public void PreservingDeclarationsOnException()
 Bang!",
                 runner.Console.Error.ToString());
         }
+
+        [Fact]
+        [WorkItem(21327, "https://github.com/dotnet/roslyn/issues/21327")]
+        public void DefaultLiteral()
+        {
+            var runner = CreateRunner(input:
+@"int i = default;
+Print(i);
+");
+            runner.RunInteractive();
+
+            AssertEx.AssertEqualToleratingWhitespaceDifferences(
+$@"Microsoft (R) Visual C# Interactive Compiler version {s_compilerVersion}
+Copyright (C) Microsoft Corporation. All rights reserved.
+
+Type ""#help"" for more information.
+> int i = default;
+> Print(i);
+0
+> ", runner.Console.Out.ToString());
+        }
+
+        [Fact]
+        [WorkItem(21327, "https://github.com/dotnet/roslyn/issues/21327")]
+        public void InferredTupleNames()
```

Comment:
I had one, but removed it on advice of Tomas. VB scripting is not sufficiently functional at this point.
The test was producing a missing type error for ValueTuple type from some CompilerServices helper type.